lib/SearchAndSort/requestUrlLimit.js

```diff
@@ -0,0 +1 @@
+export const REQUEST_URL_LIMIT = 4121;
```

Comment:
Is this value documented somewhere in Chrome docs?

Reply:
This is a server limitation according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/414). It also causes this error with the same length in Firefox. I took this number from the request URL length.